Dockerfile.windows

```diff
@@ -42,12 +42,12 @@ FROM build as windows-runtime-collect
 ARG KUBERNETES_VERSION=dev
 
 # windows runtime image
-ENV CRICTL_VERSION="v1.21.0"
-ENV CONTAINERD_VERSION="1.5.4"
+ENV CRICTL_VERSION="v1.22.0"
+ENV CONTAINERD_VERSION="1.5.5"
 ENV WINS_VERSION="0.1.1"
 ENV FLANNEL_VERSION="v0.14.0"
 ENV CALICO_VERSION="v3.19.2"
-ENV CNI_PLUGIN_VERSION="v0.9.1"
+ENV CNI_PLUGIN_VERSION="v1.0.1"
```

Comment:
Two things:
1 - After v0.9.1, cni-plugins repo does not include the flannel binary. IIRC we need the binary in case we deploy cni: canal
2 - the windows version of rke2 does not need to be hardcoded? In linux-rke2 we are using a hardened-cni-plugins version

Reply:
Canal does not support Windows and we are also not currently supporting flannel either. 

We are using upstream CNI plugins as we have no requirements for hardening or security at this time.

These binaries/versions will only ever be deployed on Windows nodes in RKE2 clusters that meets the requirement of using Calico as the CNI.

Reply:
Ok! Then no worries :)